src/workspaces/NewWorkspaceModal/NewWorkspaceWizard.tsx

```diff
@@ -0,0 +1,818 @@
+              <Select<string, true>
+                id={id}
+                isClearable={false}
+                isMulti
+                placeholder='Select groups'
+                isDisabled={!allGroups || !billingProjects}
+                value={groups}
+                onChange={(data) => {
+                  setGroups(_.map('value', data));
+                  setEnhancedBucketLogging(!!requireEnhancedBucketLogging || data.length > 0);
+                }}
+                options={_.difference(_.uniq(_.map('groupName', allGroups)), existingGroups).sort()}
+              />
+            </>
+          )}
+        </IdContainer>
+        {renderPolicyAndWorkspaceInfo()}
+      </>
+    );
+
+    // TODO can i do this prettier with Utils.cond?
+    const handleSecondaryButtonClick = async () => {
+      if (activeTab === 'basic') {
+        await create();
+      } else if (activeTab === 'sharing') {
+        setActiveTab('basic');
+      } else if (activeTab === 'security') {
+        setActiveTab('sharing');
+      }
+    };
+
+    const handlePrimaryButtonClick = async () => {
+      if (activeTab === 'basic') {
+        setActiveTab('sharing');
+      } else if (activeTab === 'sharing') {
+        setActiveTab('security');
+      } else if (activeTab === 'security') {
+        await create();
+      }
+    };
+
+    const buttons = (
+      <div
+        style={{
+          padding: '-1 -1 rem',
+          width: '90%',
+          position: 'absolute',
+          bottom: '1rem',
+          ...modalStyles.buttonRow,
+          display: 'flex',
+          justifyContent: 'space-between',
+        }}
+      >
+        <ButtonSecondary style={{ position: 'absolute', left: '1rem' }} onClick={onDismiss}>
+          Cancel
+        </ButtonSecondary>
+        <div style={{ display: 'flex', gap: '1rem', padding: '0 1rem', marginLeft: '1rem' }}>
+          <ButtonSecondary
+            style={{ padding: '0 1rem', marginLeft: '1rem', border: '1px solid', borderRadius: '5px' }}
+            disabled={activeTab === 'basic' && errors}
+            tooltip={
+              activeTab === 'basic'
+                ? 'Allows you to quickly create workspace without any sharing or additional security options'
+                : undefined
+            } // TODO should i summarize errors if there are any?  And then not show the tooltip?
+            onClick={handleSecondaryButtonClick}
+          >
+            {Utils.cond([activeTab === 'basic', () => 'Quick create workspace'], () => 'Previous')}
+          </ButtonSecondary>
+          <ButtonPrimary
+            style={{ padding: '0 1rem', marginLeft: '1rem' }}
+            disabled={activeTab === 'security' && errors}
+            tooltip={activeTab === 'security' ? Utils.summarizeErrors(errors) : undefined}
+            onClick={handlePrimaryButtonClick}
+          >
+            {Utils.cond([activeTab === 'security', () => 'Create workspace'], () => 'Next')}
+          </ButtonPrimary>
```

Comment:
Could you add Mixpanel metrics here? I'd be curious to know how many users choose "Quick create" vs "Create", and as a stretch I'd want to know how users navigate back and forth through the tabs